scripts/check-manifests.sh

```diff
@@ -1,24 +0,0 @@
-#!/bin/bash
-# Copyright 2021 NVIDIA
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# This script has been designed to run on CI only so it supposes that
```

Comment:
IIUC, this is intended to catch if changes were made that affect CRDs but were (accidently) omitted from the PR.
from the comment, this bit runs in CI.

so either:
- this is indeed no longer needed, and we should remove the check from our CI (mellanox-ci)
- this is needed and was accidently removed

Reply:
OK Checked this with @e0ne, this was used in our old travis CI, when moving to github action the check was accedently removed and he should add it back. 

please dont remove this file in this PR.